components/more-less/more-less.js

```diff
@@ -67,16 +60,8 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 			.d2l-more-less-transition {
 				transition: max-height ${transitionDur}ms cubic-bezier(0, 0.7, 0.5, 1);
 			}
-			.d2l-more-less-blur {
-				display: none;
-			}
-			:host(:not([expanded]):not([inactive])) .d2l-more-less-blur {
-				bottom: 1em;
-				content: "";
-				display: block;
-				height: 1em;
-				margin-bottom: -0.75em;
-				position: relative;
+			:host(:not([expanded]):not([inactive])) .d2l-more-less-content {
+				mask-image: linear-gradient(to top, transparent, rgb(0 0 0 / 100%) 1em);
```

Comment:
```suggestion
				-webkit-mask-image: linear-gradient(to top, transparent, #000000 1em)
```

Reply:
Assumedly we'd want both?

Reply:
Looking at the browser support (Chrome/Edge >= 120, Safari >= 15) and given that the failure scenario is simply no visible gradient when stuff gets clipped, I'd actually be OK with just the non-prefixed `mask-image` version. Or yeah, both if we really care about folks getting their gradients. 🤷

Reply:
Yep, I thought I pasted the unprefixed version as well. Fixed.

Reply:
If it were new I'd agree, but since it replaces the existing look it'd be a "regression" without it for a non-trivial percentage of users.